test/functorch/test_eager_transforms.py

```diff
@@ -1934,6 +1934,30 @@ def f(x):
                              _preallocate_and_copy=_preallocate_and_copy), chunk_size=chunk_size))(x)
         self.assertEqual(actual, expected)
 
+    @parametrize('_preallocate_and_copy', (True, False))
+    def test_chunk_jacrev_chunksize_one(self, device, _preallocate_and_copy):
+        # With chunk_size=1, we shouldn't `vmap` and hence not be limited
+        # by it's constraints.
+
+        x = torch.randn(3, device=device)
+        idx_1 = torch.tensor([0, ], device=device)
+        idx_2 = torch.tensor([0, 1], device=device)
+        chunk_size = 1
+
+        def f(x, idx):
+            # `take` doesn't work with vmap
+            # as it returns an output with dynamic shape.
+            return torch.take(x, idx)
+
+        for fn in (partial(f, idx=idx_1), partial(f, idx=idx_2)):
+            actual = jacrev(fn, chunk_size=chunk_size, _preallocate_and_copy=_preallocate_and_copy)(x,)
```

Comment:
Can we update this to jacrev(f, chunk_size=chunk_size, ...)(x, idx) so we get a test that jacrev can still accept non-Tensor inputs?